README.md

```diff
@@ -125,6 +125,90 @@ yarn test
 
 Visit http://localhost:4400/ to see the Webb Component Library! 🕸️ 🚀
 
+## Docker
+
+### Setup Container
+
+* Install and run [Docker](https://www.docker.com/)
+* Generate .env file from sample file
+```bash
+cp .env.example .env
+```
+* Run Substrate front-end from a Docker container and follow the terminal log instructions.
+```bash
+./docker-dev.sh
+```
+Note: The script requires `jq` command to be installed (i.e. Ubuntu `apt-get install -y jq`)
```

Comment:
Consider adding `wget` here, because `wget` does not exist by default in some OS.

Reply:
I don't think its a requirement as you have fallback namespaces in place, no?

Reply:
> Consider adding `wget` here, because `wget` does not exist by default in some OS.

yes, good idea. i think it would be most seamless to embed a script like this in the docker-dev.sh script https://github.com/paritytech/scripts/blob/master/get-substrate.sh, where it installs the necessary dependencies depending on what operating system it detects.

Reply:
> I don't think its a requirement as you have fallback namespaces in place, no?

i just tried a fresh install on my local macOS machine where i don't have wget and haven't created the .env file. when i run docker-dev.sh it outputs the following. so yes it uses the existing fallbacks defined in the docker-dev.sh file so it works even if wget command not found, but probably should detect their OS and try to install it.

```
./docker-dev.sh: line 7: wget: command not found
./docker-dev.sh: line 15: ./.env: No such file or directory
...
*** Started building Docker container.
*** Please wait... 
***open /Users/me/ltfschoen/webb-dapp/.env: no such file or directory
```

i was also concerned because we don't have fallbacks for the ports `PORT_BRIDGE_DAPP` and `PORT_STATS_DAPP`, but it doesn't get any further than trying to run `docker compose ...` command where expects an .env file to continue, since in docker-compose-dev.yml it has:
```
services:
  dev:
    ...
    env_file:
      - .env
```

i think we'd only be concerned about fallbacks if the user created an empty .env file but didn't follow the README.md instructions where it says to create it based on the .env.example with `cp .env.example .env`